### ISSUE TITLE
Bug: Failed file moves need error handling

### ISSUE THREAD

**Issue by TheTaylorLee** [TheTaylorLee/docker-transcodeautomation#29]
**Environment**
- Image Tag/Build: 2.10.1

**Describe the bug**
Sometimes Move-FileToMediaFolder function will fail. The cause is not tracked, but I think it likely happens because sometimes a copy job can be interrupted. This will result in the target file not existing and erroring out.

**Expected behavior**
To either have the copy job fail or build in some error handling for this specific scenario.

**Logs**
```log
Get-ChildItem: /docker-transcodeautomation/functions/Move-FileToMediaFolder.ps1:52
Line |
  52 |              $oldsizemb = (Get-ChildItem $destination.fullname | Selec …
     |                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path '/media/Shows/filename.mkv' because it does not exist.

Get-ChildItem: /docker-transcodeautomation/functions/Move-FileToMediaFolder.ps1:52
Line |
  52 |              $oldsizemb = (Get-ChildItem $destination.fullname | Selec …
     |                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path '/media/m4/Shows/filename.mkv' because it does not exist.
```

**Comment by TheTaylorLee** [comment]
Line 52 will fail to pull the old size and result in the try/catch failing.

```powershell
$oldsizemb = (Get-ChildItem $destination.fullname | Select-Object @{ Name = "oldsizeMB"; Expression = { [math]::round(($_.length / 1mb), 2) } }).oldsizeMB
```

Can resolve this by excluding that statistic, or populating that value from another source. Either use a conditional parameter without erroring out or move the try catch block.

**Comment by TheTaylorLee** [comment]
Resolved by moving failed file moves error handling into an else block. I also added a call for the Move-FileToMediaFolder function to the initial Invoke-TranscodeAutomation entrypoint script. This ensures that each processing period will run this function regardless of if any new files are to be transcoded. As a result ensures that the error is handled sooner rather. A power failure event for example would result in this being the only way to get this error handled somewhat quickly. So I consider it the best option.
